components/Calendar.tsx

```diff
@@ -38,13 +39,13 @@ export default function Calendar({ className = '', size }: ICalendarProps) {
           <li key={index} data-testid='Calendar-list-item'>
             <a href={event.url} className='mb-1 mt-2 flex grow flex-col items-start sm:flex-row sm:items-center'>
               <div className='inline-flex h-12 min-w-12 flex-row rounded-full bg-pink-500 font-bold text-white'>
-                <span className='flex-1 self-center text-center'>{event.date.format('D')}</span>
+                <span className='flex-1 self-center text-center'>{moment(event.date).format('D')}</span>
```

Comment:
Why have you used the `moment` library here?

Reply:
`event.date` was of type string, which doesn't have a format method. So, I have first converted it into a date and then used format method.